cypress/integration/malicious-proposal-validation.spec.ts

```diff
@@ -0,0 +1,45 @@
+/// <reference path="../support/index.d.ts" />
+
+import { EPOCH_LENGTH } from '../support/common';
+
+describe('Malicious proposal validation', () => {
+  it('shows a warning if the original EVM script and the decoded EVM script do not match', () => {
+    cy.increaseTimeAndRelogin(EPOCH_LENGTH + 60 * 60); // skip the genesis epoch (add 1 hour just to be sure)
+    /*
```

Comment:
@mcoetzee btw. is this comment still valid? I guess you changed this after Buraks suggestion

Reply:
Yes this is still valid 👍 (a new test was added based on Burak's feedback)